deploy/olm-catalog/kogito-operator/0.13.0/kogito-operator.v0.13.0.clusterserviceversion.yaml

```diff
@@ -4,21 +4,6 @@ metadata:
   annotations:
     alm-examples: |-
       [
-        {
```

Comment:
This file shouldn't be updated, but it was since we don't have 0.14 yet. As soon as we bump, we will need to rebase this PR for this changes to impact only 0.14.x.

Reply:
done

Reply:
This file continues here, we should not merge with this one.

Reply:
File comming in changelist due to auto formating issue with Intellij IDE. I have removed this file now